fork-visualizer/src/app/components/HorizontalBlockList.jsx

```diff
@@ -0,0 +1,95 @@
+"use client";
+
+import { useEffect, useRef, useState } from 'react'
+import { AUTO_SCROLL_DEFAULT } from "@/constants"
+
+const Block = ({ node }) => {
+    return (
+        <div className={`p-4 w-32 h-32 col-span-1 m-4 overflow-scroll ${ node.duplicate ? "border-8 border-rose-900" : "" } rounded-lg ${node.groupColor}`} id={node.hash}>
```

Comment:
```suggestion
        <div className={`p-4 w-32 h-32 col-span-1 m-4 overflow-scroll ${ node?.duplicate ? "border-8 border-rose-900" : "" } rounded-lg ${node?.groupColor}`} id={node?.hash}>
```

Reply:
I have removed this file because it is no longer used